e3db-crypto-android/src/main/java/com/tozny/e3db/crypto/FileSystemManager.java

```diff
@@ -0,0 +1,60 @@
+package com.tozny.e3db.crypto;
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * 
+ * Copyright (c) 2018 
+ * 
+ * All rights reserved.
+ * 
+ * e3db-java
+ * 
+ * Created by Lilli Szafranski on 4/9/18.
+ * 
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+
+import android.content.Context;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+
+public class FileSystemManager {
```

Comment:
I don't think the checked exceptions add any value on the public methods here. If there isn't any reasonable recovery (example, filesystem can't be found) then there isn't any reason to expose these to the programmer. I'd rather they were RuntimeExceptions or similar.